server/container/guice/mailbox/pom.xml

```diff
@@ -44,6 +44,10 @@
             <groupId>${james.groupId}</groupId>
             <artifactId>apache-james-mailbox-store</artifactId>
         </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
```

Comment:
it doesn't make sense for any kind of mailbox thus It should not be in guice-mailbox